### DISCUSSION TITLE
Welcome to docker-handbrake Discussions!

### DISCUSSION THREAD

**Discussion by jlesage** [jlesage/docker-handbrake#270] [Announcements]
## 👋 Welcome!
Discussions has been enabled for this project.  This is a place to connect with other members of the community.

Here, you can:
  * Ask questions about:
    * How to setup the container.
    * How to use the application.
    * Problems you might with your specific setup.
    * etc.
  * Share ideas.
  * Engage with other community members.

Welcome others and be open-minded. Remember, this is a community we build together 💪.

**Comment by vdrover** [comment]
Nice work!

**Comment by alexindigo** [comment]
Exciting!
